cluster/gh-action-runner/gh-action-runner-template-create.yml

```diff
@@ -0,0 +1,262 @@
+        add_host:
+          hostname: '{{ collected_address }}'
+          name: test-vm
+          ansible_ssh_host: '{{ collected_address }}'
+          ansible_ssh_user: root    
+          ansible_ssh_common_args: '-F /tmp/ssh-conf'      
+        tags:
+          - always         
+      - name: check connectivity to the VM
+        ping:
+        delegate_to: test-vm
+
+      - name: resize the root partition 
+        shell: |
+          growpart  /dev/vda 1
+          resize2fs /dev/vda1
+        delegate_to: test-vm
+
+      - name: upgrade to fedora-36
+        block:
+        - name: install system upgrade plugin
+          dnf: name="dnf-plugin-system-upgrade" state=present  
+        - name: execute upgrade in system-upgrade mode
+          shell: dnf system-upgrade download --releasever=36 -y
+#           dnf system-upgrade download -y --allowerasing --best --releasever 36
+        - name: run system-upgrade reboot command
+          shell: 'sleep 2 && dnf system-upgrade reboot -y'
+          ignore_errors: true
+          async: 1
+          poll: 0
+        - name: wait for server to come back after reboot
+          wait_for_connection:
+        delegate_to: test-vm
+
+      - name: install pre-req
+        block:
+        - name: install-dev-tools
+          dnf: name="@Development tools" state=present       
+        - name: Development Libraries
+          dnf: name="@Development Libraries" state=present  
+        - name: Docker | CE | DNF | Deploy repository (systems without SNI)
+          shell: "dnf config-manager --add-repo {{ docker_ce_repo_file_url }}"
+          args:
+            creates: /etc/yum.repos.d/docker-ce.repo
+        - name: enable bazel repo
+          shell: "dnf copr enable vbatts/bazel -y"            
+        - name: install packages
+          dnf: 
+            name:
+              - git
+              - wget
+              - make
+              - glibc-static
+              - libstdc++-static
+              - docker-ce
+              - bazel5
+            state: latest
+
+        - name: start docker
+          systemd:
+            name: docker
+            state: started
+            enabled: true
+        - ansible.posix.sysctl:
+            name: fs.inotify.max_user_instances
+            value: '512'
+            sysctl_set: true
+            state: present            
+        - name: install golang
+          shell: |
+            wget https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
+            tar -C /usr/local -xzf go1.20.4.linux-amd64.tar.gz
+
+        - name: install kubectl
+          shell: |
+            curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl
+            chmod +x /usr/local/bin/kubectl
+
+        - name: install k9s
+          shell: |
+            curl -L https://github.com/derailed/k9s/releases/download/v0.27.4/k9s_Linux_amd64.tar.gz -o /tmp/k9s.tar.gz
```

Comment:
Hmm would probably also move this to some variable so we don't need to go through code to know which version do we install